alpha/books/RayTracingInOneWeekend.html

```diff
@@ -3737,10 +3758,10 @@
 
 The reason we have defocus blur in real cameras is because they need a big hole (rather than just a
   pinhole) through which to gather light.
-A large hole would defocus everything, but if we stick a lens in the hole, there will be a certain
-  distance where everything is in focus.
-Objects appear blurrier the further they are from that distance (either closer or further from the
-  camera).
+A large hole would defocus everything, but if we stick a lens in front of the sensor, there will be
+a certain distance at which everything is in focus.
```

Comment:
Oof. I hate that we mention a sensor, but never define it what that means and then continue with a model that doesn't have one anyway. I also wish we could drop the concept of a lens entirely, as we don't have that either (or any object that bends light rays), and say that having one must imply "a certain distance at which everything is in focus" as if we just proved something. Sigh; moving along.

Reply:
I clocked this as well

I just dropped it as I didn't think we had time, nor was that a string I wanted to pull on.

I'll see if there's anything small worth doing